mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java

```diff
@@ -186,42 +183,19 @@ void registerGroupShouldCreateRetryExchange() throws Exception {
     }
 
     @Test
-    void rabbitMQChannelPoolDoesntLimitNumberOfChannelsCreated() {
+    void rabbitMQChannelPoolShouldThrowWhenTimeoutAndAllChannelsAreTaken() {
         Mono<? extends Channel> channelMono = eventBus.getChannelPool()
             .getChannelMono();
 
         assertThat(eventBus.getChannelPool().getChannelsQueueCapacity())
             .isEqualTo(5);
 
-        int channel1Number = channelMono.block().getChannelNumber();
-        int channel2Number = channelMono.block().getChannelNumber();
-        int channel3Number = channelMono.block().getChannelNumber();
-        int channel4Number = channelMono.block().getChannelNumber();
-        int channel5Number = channelMono.block().getChannelNumber();
-        int channel6Number = channelMono.block().getChannelNumber();
-        ImmutableList<Integer> channelNumberList = ImmutableList.of(channel1Number, channel2Number, channel3Number,
-            channel4Number, channel5Number, channel6Number);
-
-        assertThat(channelNumberList)
-            .hasSize(6)
-            .doesNotHaveDuplicates();
-    }
-
-    @Test
-    void concurrentRequestOnChannelMonoLeadToChannelsHungryException() {
```

Comment:
I'd like to keep that test and know our behaviour under "floods"

Reply:
you got it, checkout the PR commits